### DISCUSSION TITLE
QUESTION | log的err！是不是改为 FATAL 更为直观？

### DISCUSSION THREAD

**Discussion by StarryCamile** [MCSLTeam/MCSL2#81] [提问区域]
- [x] 未在 [Issues](https://github.com/LxHTT/MCSL2/issues) 中看到其他相同疑问
- [x] MCSL 2 运行在受官方支持的平台
___

**Comment by lxhtt** [comment]
是（  
但是日志前缀不对齐难受啊

**Comment by lxhtt** [comment]
所以如果还有别的词语就好了（（
